operator/v1/types_console.go

```diff
@@ -146,6 +146,11 @@ type ConsoleCustomization struct {
 	// +listMapKey=id
 	// +optional
 	Perspectives []Perspective `json:"perspectives"`
+	// ingress allows to configure the alternative ingress for the console.
+	// This field is intended for clusters without ingress capability,
+	// where access to routes is not possible.
+	// +optional
+	Ingress IngressCustomization `json:"ingress,omitempty"`
```

Comment:
Sorry I've missed this one.
The `Ingress` field should go directly under the `ConsoleSpec`, not under `Customization`. In `Customization` we keep the UI related customizations.

Reply:
Done.

Setup of the alternative ingress URLs for the console looks like this now:
```
spec:
  ingress:
    consoleURL: https://console-alb.com
    clientDownloadsURL: https://downloads-alb.com
```